Sources/Pocket/Pocket+Authentication.swift

```diff
@@ -13,7 +13,8 @@ import Foundation
 extension Pocket {
 
     public static let requestTokenUrl = URL(string: "https://getpocket.com/v3/oauth/request")!
-    public static let authorizeUrl = URL(string: "https://getpocket.com/v3/oauth/authorize")!
+    public static let authorizeUrl = URL(string: "https://getpocket.com/v3/auth/authorize")!
```

Comment:
If I see correctly, this is not scoped to the versioned API at all.
```suggestion
    public static let authorizeUrl = URL(string: "https://getpocket.com/auth/authorize")!
```